wrappers/ios/ci/build.sh

```diff
@@ -15,7 +15,8 @@ for i in $(ls -t $OPENSSL_PATH); do
     break
 done
 
-INDY_VERSION="v1.16.0"
+#INDY_VERSION="efb7215" # this is "v1.16.0" + rusql update fix + (number of other commits on master branch)
+INDY_VERSION="b4b330ef3" # this is "v1.16.0" tag
```

Comment:
What is the purpose of changing this from `v1.16.0` to the tag `v1.16.0` commit?

Reply:
Experiment ^_^